src/NServiceBus.Core/Sagas/CustomFinderAdapter.cs

```diff
@@ -1,8 +1,7 @@
-namespace NServiceBus.Sagas
+namespace NServiceBus.Saga
```

Comment:
I agree but all public classes are currently siting in Saga so moving all
those to sagas is a much bigger change

On Friday, March 27, 2015, Andreas Öhlund <notifications@github.com> wrote:

> In src/NServiceBus.Core/Sagas/CustomFinderAdapter.cs
> <https://github.com/Particular/NServiceBus/pull/2693#discussion_r27279193>
> :
>
> > @@ -1,8 +1,7 @@
> > -namespace NServiceBus.Sagas
> > +namespace NServiceBus.Saga
>
> Makes alot of sense. But I think Sagas is a better namespace name?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/2693/files#r27279193>.
>